tests/cpi-returns/programs/callee/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+use anchor_lang::prelude::*;
+
+declare_id!("Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS");
+
+#[program]
+pub mod callee {
+    use super::*;
+
+    #[derive(AnchorSerialize, AnchorDeserialize)]
+    pub struct StructReturn {
+        pub value: u64,
+    }
+
+    pub fn initialize(_ctx: Context<Initialize>) -> Result<()> {
+        Ok(())
+    }
+
+    pub fn return_u64(_ctx: Context<CpiReturn>) -> Result<u64> {
+        Ok(10)
+    }
+
+    pub fn return_struct(_ctx: Context<CpiReturn>) -> Result<StructReturn> {
+        let s = StructReturn { value: 11 };
+        Ok(s)
+    }
```

Comment:
would be good to test a `Vec` too if that's even possible to return

Reply:
👍